source/includes/fundamentals/code-snippets/records/NetworkDeviceRecord.java

```diff
@@ -0,0 +1,15 @@
+package fundamentals.codecs.records;
+
+// start networkDeviceRecord
+import org.bson.BsonType;
+import org.bson.codecs.pojo.annotations.BsonProperty;
+import org.bson.codecs.pojo.annotations.BsonId;
+import org.bson.codecs.pojo.annotations.BsonRepresentation;
+
+public record NetworkDeviceRecord(
+        @BsonId() @BsonRepresentation(BsonType.OBJECT_ID) String deviceId,
```

Comment:
Nit: might be easier to read if each annotation was on its own line.  That's usually how I write them